source/extensions/filters/http/transformation/transformation_filter_config.h

```diff
@@ -50,21 +43,26 @@ class TransformationFilterConfig : public FilterConfig {
   TransformationFilterConfig(const TransformationConfigProto &proto_config,
                              const std::string &prefix, Server::Configuration::FactoryContext &context);
 
-  const std::vector<MatcherTransformerPair> &transformerPairs() const override {
-    return transformer_pairs_;
-  };
-
   std::string name() const override {
     return SoloHttpFilterNames::get().Transformation;
   }
 
   bool logRequestResponseInfo() const {
     return log_request_response_info_;
   }
+protected:
+
+  const std::vector<MatcherTransformerPair> &transformerPairs() const override {
```

Comment:
why does this need to be protected?

Reply:
it was public before and it confused me, so I moved to protected so its more clear it is not meant to be used by non sub-classes